.github/workflows/release.yml

```
@@ -165,11 +165,57 @@ jobs:
           cp scripts/install.sh pages/
           cp .github/logo.svg pages/
 
+          # Copy SEO files
+          echo "📄 Copying SEO files..."
+          cp .github/robots.txt pages/
+          cp .github/sitemap.xml pages/
+          cp .github/manifest.json pages/
+          cp .github/browserconfig.xml pages/
+          cp .github/humans.txt pages/
+          cp .github/schema.json pages/
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)